code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -26,10 +26,45 @@
 	window_name = "Automatic Medical Unit v1.1"
 	data_hud_type = DATA_HUD_MEDICAL_ADVANCED
 
+	var/list/idle_phrases = list(
+		MEDIBOT_VOICED_MASK_ON = 'sound/voice/mradar.ogg',
+		MEDIBOT_VOICED_ALWAYS_A_CATCH = 'sound/voice/mcatch.ogg',
+		MEDIBOT_VOICED_PLASTIC_SURGEON = 'sound/voice/msurgeon.ogg',
+		MEDIBOT_VOICED_LIKE_FLIES = 'sound/voice/mflies.ogg',
+		MEDIBOT_VOICED_DELICIOUS = 'sound/voice/mdelicious.ogg',
+	)
+
+	var/list/finish_healing_phrases = list(
+		MEDIBOT_VOICED_ALL_PATCHED_UP = 'sound/voice/mpatchedup.ogg',
+		MEDIBOT_VOICED_APPLE_A_DAY = 'sound/voice/mapple.ogg',
+		MEDIBOT_VOICED_FEEL_BETTER = 'sound/voice/mfeelbetter.ogg',
+	)
+
+	var/list/located_patient_phrases = list(
+		MEDIBOT_VOICED_HOLD_ON = 'sound/voice/mcoming.ogg',
+		MEDIBOT_VOICED_WANT_TO_HELP = 'sound/voice/mhelp.ogg',
+		MEDIBOT_VOICED_YOU_ARE_INJURED = 'sound/voice/minjured.ogg',
+	)
+
+	var/list/patient_died_phrases = list(
+		MEDIBOT_VOICED_STAY_WITH_ME = 'sound/voice/mno.ogg',
+		MEDIBOT_VOICED_LIVE = 'sound/voice/mlive.ogg',
+		MEDIBOT_VOICED_NEVER_LOST = 'sound/voice/mlost.ogg',
+	)
+
+	var/list/frustration_phrases = list(
+		MEDIBOT_VOICED_FUCK_YOU = 'sound/voice/mfuck_you.ogg',
+	)
+
+	/// The above lists joined into one, for one-place lookups
+	var/list/all_phrases
```

Comment:
Please make these into a nested global list similarly to `footstep_defines.dm`

Reply:
theyre scoped so you can add funny messages to subtypes.